paasta_tools/setup_marathon_job.py

```diff
@@ -392,6 +399,17 @@ def log_bounce_action(line: str, level: str='debug') -> None:
                 ),
                 level='event',
             )
+
+            if yelp_meteorite:
+                # Emit deploy.paasta event
```

Comment:
This comment doesn't really say anything that the next line doesn't say

Reply:
Agreed. Will remove this